source/command-monitoring/tests/unified/find.yml

```diff
@@ -0,0 +1,290 @@
+description: "find"
+
+schemaVersion: "1.1"
+
+createEntities:
+  - client:
+      id: &client client
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+        - commandFailedEvent
+  - database:
+      id: &database database
+      client: *client
+      databaseName: &databaseName command-monitoring-tests
+  - collection:
+      id: &collection collection
+      database: *database
+      collectionName: &collectionName test
+
+_yamlAnchors:
+  namespace: &namespace "command-monitoring-tests.test"
+
+initialData:
+  - collectionName: *collectionName
+    databaseName: *databaseName
+    documents:
+      - { _id: 1, x: 11 }
+      - { _id: 2, x: 22 }
+      - { _id: 3, x: 33 }
+      - { _id: 4, x: 44 }
+      - { _id: 5, x: 55 }
+
+tests:
+  - description: "A successful find with no options"
+    operations:
+      - name: find
+        object: *collection
+        arguments:
+          filter: { _id: 1 }
+    expectEvents:
+      - client: *client
+        events:
+          - commandStartedEvent:
+              command:
+                find: *collectionName
+                filter: { _id: 1 }
+              commandName: find
+              databaseName: *databaseName
+          - commandSucceededEvent:
+              reply:
+                ok: 1
+                cursor:
+                  id: 0
+                  ns: *namespace
+                  firstBatch:
+                    -  { _id: 1, x: 11 }
+              commandName: find
+
+  - description: "A successful find with options"
+    operations:
+      - name: find
+        object: *collection
+        arguments:
+          filter: { _id: { $gt: 1 } }
+          sort: { _id: 1 }
+          skip: 2
+          comment: "test"
+          hint: { _id: 1 }
+          max: { _id: 6 }
+          maxTimeMS: 6000
+          min: { _id: 0 }
+          returnKey: false
+          showRecordId: false
+          readPreference:
+            mode: primaryPreferred
```

Comment:
I think the old test was maybe meant to cover this language from the section linked above: 
> The read preference will subsequently be dropped as it is considered metadata and metadata is not currently provided in the command events.

If so though, it would probably make sense to assert here though that the read preference is unset in the command.

similarly to above we could use a separate collection if we want to avoid per-operation read preference